src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/CpsPackageReferenceProject.cs

```diff
@@ -116,6 +116,13 @@ private PackageSpec GetPackageSpec()
             return null;
         }
 
+        /// <inheritdoc/>
+        internal override ValueTask<PackageSpec> GetPackageSpecAsync(CancellationToken ct)
```

Comment:
If you're only going to create a new ValueTask, you're not really getting the benefit of ValueTask. 

In fact, ValueTask allocates more in this case, it's larger than Task by default and you're not really having enough ValueTask methods where reusing can make a dent.

Reply:
Changed to Task.